pkg/solana/logpoller/query.go

```diff
@@ -0,0 +1,126 @@
+package logpoller
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+)
```

Comment:
While reviewing @EasterTheBunny 's DSL Parsing PR, I noticed we're missing unit tests for the functions in this file.  In evm we have a `Test_QueryArgs` set of tests that runs through these in a `query_test.go` file. Should we add something similar here?

Reply:
Replied in the DSL Parsing PR https://github.com/smartcontractkit/chainlink-solana/pull/959/files#r1876556494